README.md

```diff
@@ -162,7 +162,6 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Actions for Deno](https://github.com/denolib/setup-deno)
 - [GitHub Actions for Unity](https://github.com/webbertakken/unity-actions)
 - [Octions - GitHub Actions for GitHub REST API](https://github.com/maxkomarychev/octions)
-- [GitHub Actions for Docker](https://github.com/docker/github-actions)
```

Comment:
Please don't remove this from the official GitHub section

Reply:
https://github.com/docker/github-actions is deprecated that's why. What is the constraint?